### PR TITLE
Tuedontuoyo/get 18 navigation and desktop layout for

### PR DESCRIPTION
**Pull request by 0tuedon**
I design the landing page as me and @asapconet  discuss if there is any other thing you would like me to add please do. 
Screenshot (265)
The above is the design for the landing page coded and responsive

### CODE REVIEW

**Comment by 0tuedon** [comment]
Screenshot (270)
The new Ui

**Comment by Eronmmer** [comment]
Screenshot 2022-01-02 at 22 25 47

@0tuedon please be consistent with casing for outer-level folders
All the folders under another folder or directory should use the same case. In the screenshot above, `Components`, `Pages`, and `utils` should either be in lowercase or uppercase

**Comment by Eronmmer** [comment]
Screenshot 2022-01-02 at 22 35 25

An image shouldn't be in a `utils` folder. Move all images to `/public/assets`. 
Ideally, a `utils` folder should contain utility, reusable functions, constants, classes, and other blocks of code that can be reused  more than once in the codebase

**Comment by Eronmmer** [comment]
Screenshot 2022-01-02 at 22 43 21

Some suggestions
- Increase the border of the input element here
- Add some margin between the input element and the button
- Give the button some life
  - Add more padding
  - Change the color to something like yellow?
- Add some margin after the 'Get Early Access' input label

**Comment by Eronmmer** [comment]
Screenshot 2022-01-02 at 22 47 55

@0tuedon  I think you should add some padding to the navigation links. You may want to reduce the margin between the two social icons on the top right. You may also want to increase the font size of the 'GetnShare' header on the top left

File: src/Components/Button.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Button = ({ name }) => {
+  return (
+    <div className="  lg:mr-4 text-black">
+      <button className="bg-white  shadow-md px-9  hover:bg-amber-400 py-4 rounded-3xl border-slate-400">
```
Comment (Eronmmer) [inline src/Components/Button.js:6]: @0tuedon @asapconet, you can check the comments I dropped in this PR for the requested changes

**Review by Eronmmer — CHANGES_REQUESTED** [review]

**Comment by 0tuedon** [comment]
Screenshot (281)
Folder structure done
Screenshot (282)
And i have  reworked the ui of the landing page 
@Eronmmer

**Comment by 0tuedon** [comment]
> Screenshot 2022-01-02 at 22 47 55
> 
> @0tuedon I think you should add some padding to the navigation links. You may want to reduce the margin between the two social icons on the top right. You may also want to increase the font size of the 'GetnShare' header on the top left

Okay will do

**Comment by 0tuedon** [comment]
> Screenshot 2022-01-02 at 22 47 55
> 
> @0tuedon I think you should add some padding to the navigation links. You may want to reduce the margin between the two social icons on the top right. You may also want to increase the font size of the 'GetnShare' header on the top left

Screenshot (284)
This has been done check the lastest commit

**Review by Eronmmer — APPROVED** [review]

**Comment by 0tuedon** [comment]
@asapconet  how is this design
Screenshot (288)

**Comment by 0tuedon** [comment]
Please also @Eronmmer  feel free to say what i should change

**Pull request merged**